apps/qubit/modules/informationobject/actions/renameAction.class.php

```diff
@@ -22,39 +22,28 @@ class InformationObjectRenameAction extends sfAction
   // Allow modification of title, slug, and digital object filename
   public function execute($request)
   {
-    // Return 401 if unauthorized
-    if (!sfContext::getInstance()->user->isAuthenticated()
-      || !QubitAcl::check($this->resource, 'update'))
-    {
-      $this->response->setStatusCode(401);
-      return sfView::NONE;
-    }
-
```

Comment:
I think it's a good idea to keep the ACL check, to avoid manually modified requests.